### DISCUSSION TITLE
How can I fix this installation error on Debian 12 ?

### DISCUSSION THREAD

**Discussion by dogognon** [motis-project/motis#377] [Q&A] [answered]
What are the prerequisites for installing Motis on new debian install ?
![capt git](https://github.com/motis-project/motis/assets/101815673/28e6905a-05ea-428a-b007-8a035413c42f)

**Comment by felixguendling** [comment]
Did you try the "no-avx" version?

**Reply by dogognon** [reply]
no.I just ran the Linux script from the wiki page on Debian in VirtualBox.

**Reply by dogognon** [reply]
How can I do that?

**Reply by felixguendling** [reply, marked as answer]
You can replace the MOTIS distribution with this release:
https://github.com/motis-project/motis/releases/download/v0.11.3/motis-linux-amd64-noavx.tar.bz2

**Reply by dogognon** [reply]
Thank you.
